sugarcrm/client.py

```diff
@@ -0,0 +1,34 @@
+import sugarcrm
+from sugarcrm.api import SugarCrmAPI
+
+
+class Client:
```

Comment:
I find it a bit confusing for this class to be named Client. I expect a client to be dealing with transport-level concerns and not business-level ones.

Reply:
Actually I could rename the api.py to client.py and this client.py and as you mention in the other comment, encapsulate most of this logic into a Customer class in a customer.py file. 
Then we could mix the authenttication line for the API, the get customer object by email's line and finding if it has membership role into the task.py file itself. 

What do you think?